src/main/resources/docs/description/description.json

```diff
@@ -101,13 +101,6 @@
 }, {
   "patternId" : "category_java_errorprone_UselessOperationOnImmutable",
   "title" : "An operation on an Immutable object (String, BigDecimal or BigInteger) won't change the object itself"
-}, {
```

Comment:
The DocGenerator should sort this file by `patternId` so we get stable diffs.

Reply:
that would be nice